### PR TITLE
Fix wb_supervisor_add_force_with_offset()

### PR DESCRIPTION
**Pull request by fabienrohrer**
The offset is wrongly computed, probably because of the ODE rotation which is not correctly taken into account. Using `dBodyAddForceAtPos()` instead of `dBodyAddForceAtRelPos()` simplifies and fixes this issue.

### CODE REVIEW

**Comment by fabienrohrer** [comment]
[✔️](https://travis-ci.com/cyberbotics/webots/builds/143698198)👍

File: docs/reference/changelog-r2020.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Released on XXX YYYrd, 2020.
     - Improved the SUMO interface to use the version of SUMO already installed (if any).
     - Improved the SUMO interface to be compatible with the latest version (1.4).
   - Bug fixes
+    - Fix the [`wb_supervisor_node_add_force_with_offset()`](supervisor.md#wb_supervisor_node_add_force_with_offset) function.
```
Comment (DavidMansolino) [inline docs/reference/changelog-r2020.md]:
```suggestion
    - Fixed the direction of the force applied with the [`wb_supervisor_node_add_force_with_offset()`](supervisor.md#wb_supervisor_node_add_force_with_offset) function.
```

File: src/webots/nodes/utils/WbSupervisorUtilities.cpp [hunk 2]
```diff
@@ -794,13 +794,17 @@ void WbSupervisorUtilities::handleMessage(QDataStream &stream) {
       WbNode *const node = getProtoParameterNodeInstance(WbNode::findNode(id));
       WbSolid *const solid = dynamic_cast<WbSolid *>(node);
       if (solid) {
-        WbVector3 force(fx, fy, fz), offset(ox, oy, oz);
+        const WbMatrix4 &solidMatrix = solid->matrix();
+
+        WbVector3 offset = solidMatrix * WbVector3(ox, oy, oz);
```
Comment (omichel) [inline src/webots/nodes/utils/WbSupervisorUtilities.cpp]:
```suggestion
        const WbVector3 offset = solidMatrix * WbVector3(ox, oy, oz);
```

**Comment by fabienrohrer** [comment]
👍

**Comment by fabienrohrer** [comment]
> Thank you, this seems much cleaner, but what about the C_SUPERVISOR_NODE_ADD_FORCE case?

Just saw this comment.. This case should work smoothly, the computation of the offset is much simpler in this case and seems to be correctly done.

**Pull request merged**